meson.build

```diff
@@ -0,0 +1,323 @@
+project('notcurses', ['c', 'cpp'],
+        default_options:
+          ['c_std=gnu11',
+           'cpp_std=gnu++17',
+           'default_library=both'],
+        version: '2.2.4',
+        license: 'Apache-2.0',
+        meson_version: '>=0.57.1')
+
+fs = import('fs')
+
+cc = meson.get_compiler('c')
+cxx = meson.get_compiler('cpp')
+
+compflags = ['-Wshadow','-Wformat','-fexceptions', '-Og', '-D_FORTIFY_SOURCE=2']
```

Comment:
Please remove `-Og` and `-D_FORTIFY_SOURCE=2`. As mentioned earlier they should not be set from within `meson.build` files.